sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/AzMonList.cs

```diff
@@ -81,7 +81,7 @@ internal static object[] GetTagValues(ref AzMonList list, params string[] tagNam
         {
             int lengthTagNames = tagNames.Length;
             int lengthList = list.Length;
-            object[] values = new object[(int)lengthTagNames];
+            object[] values = new object[lengthTagNames];
```

Comment:
`lengthTagNames` is already an int and doesn't need the cast. We should invest in an Analyzer to catch things like this.